### DISCUSSION TITLE
Pesquisa sobre acessibilidade web

### DISCUSSION THREAD

**Discussion by brunopulis** [frontendbr/forum#280] [Discussão]
Fala pessoal, tudo certo? 

Estou fazendo uma pesquisa sobre Acessibilidade Web e gostaria que pudessem me ajudar.
Segue o link da pesquisa. 

[Pesquisa sobre acessibilidade](https://goo.gl/forms/ZLI7FTUHeDsyZ7C22)

**Comment by vitor-mendrone-deel** [comment]
Respondida!

**Comment by fabiootc** [comment]
Respondida.

**Comment by afonsopacifer** [comment]
Respondida ;)

**Comment by felipefialho** [comment]
Done!

**Comment by gabrieldarezzo** [comment]
Respondida xD

**Comment by andrewsfg** [comment]
Respondida :D

**Comment by gss-patricia** [comment]
Feito!
